Src/FluentAssertions/Common/TypeExtensions.cs

```diff
@@ -134,8 +134,15 @@ public static bool IsEquivalentTo(this IMember property, IMember otherProperty)
                    property.Name == otherProperty.Name;
         }
 
+        private static readonly Type[] EmptyTypeArray = new Type[0];
+
         public static Type[] GetClosedGenericInterfaces(Type type, Type openGenericType)
         {
+            if (Type.GetTypeCode(type) != TypeCode.Object)
+            {
+                return EmptyTypeArray;
```

Comment:
Why not using `Array.Empty<Type>()`?
I think this is using a singleton, too.

Reply:
Is that available on all the target platforms?

Reply:
I think so, everything since .net framework 4.6
https://docs.microsoft.com/en-us/dotnet/api/system.array.empty

Reply:
:+1: